pkg/machine/shim/host.go

```diff
@@ -196,30 +196,33 @@ func Init(opts machineDefine.InitOptions, mp vmconfigs.VMProvider) error {
 	// copy it into the conf dir
 	if len(opts.IgnitionPath) > 0 {
```

Comment:
should we just create a new function to deal with this given the more complexity here?  wdyt @luap99

Reply:
The function is pretty long already so splitting it up makes sense but I would not force a community contributor to do that work. It only adds one more if/else here so I don't think it is required to rework just for that.

Reply:
im cool with that